ydb/docs/en/core/integrations/migration/dbt.md

```diff
@@ -0,0 +1,245 @@
+
+   | Parameter             | Required | Default Value | Description                                                                 |
+   |-----------------------|----------|---------------|-----------------------------------------------------------------------------|
+   | incremental_strategy  | No       | MERGE         | [Incremental materialization strategy](https://docs.getdbt.com/docs/build/incremental-strategy). The MERGE strategy is supported, using the {{ydb}} [UPSERT](../../yql/reference/syntax//upsert_into.md) operation. APPEND strategy support is under development. |
+
+{% note info %}
+
+Another materialization type, [ephemeral model](https://docs.getdbt.com/docs/build/materializations#ephemeral), is not currently supported by the connector.
+
+{% endnote %}
+
+### Snapshots
+
+The [snapshot](https://docs.getdbt.com/docs/build/snapshots) mechanism is not currently supported by {{ dbt-ydb }}.
+
+### Seeds (CSV-based reference/test data)
+
+The {{ dbt-ydb }} connector supports {{dbt}} [seeds](https://docs.getdbt.com/docs/build/seeds) for upload reference and test data from CSV files into your project and using them in other models.
+
+### Data Testing
+
+{{ dbt-ydb }} supports standard [dbt data tests](https://docs.getdbt.com/docs/build/data-tests#generic-data-tests), as well as [singular tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests) within the capabilities of [YQL](../../yql/reference/index.md).
+
+### Documentation Generation
+
+{{ dbt-ydb }} supports generating [documentation](https://docs.getdbt.com/docs/build/documentation) from {{dbt}} projects for {{ydb}}.
+
+## Getting Started
+
+### Requirements
+
+To start working with {{dbt}} on {{ydb}}, you will need:
+
+- Python 3.10+;
+- dbt Core (1.8+);
+- An existing {{ ydb-short-name }} cluster (a single-node installation from the [quickstart](../../quickstart.md) is sufficient).
+
+{% note info %}
+
+Note! dbt Fusion (2.0) is not supported at this time.
+
+{% endnote %}
+
+### Installation
+
+To install {{ dbt-ydb }}, run:
+
+```bash
+pip install dbt-ydb
+```
+
+### Connecting {{dbt}} to a {{ydb}} Cluster
+
+{{dbt}} connects to {{ydb}} through the {{dbt-ydb}} connector in the [standard way](../../concepts/connect.md) for {{ydb}}. To connect successfully, you need to specify the endpoint, database path, and authentication parameters in the dbt [profiles](https://docs.getdbt.com/docs/core/connect-data-platform/connection-profiles) file.
+
+Example profile file with possible authentication options and default values (in square brackets):
+
+```yml
+profile_name:
+target: dev
+outputs:
+   dev:
+      type: ydb
+      host: [localhost] # YDB host
+      port: [2136] # YDB port
+      database: [/local] # YDB database
+      schema: [<empty string>] # Optional subfolder for DBT models
+      secure: [False] # If enabled, grpcs protocol will be used
+      root_certificates_path: [<empty string>] # Optional path to root certificates file

```

Comment:
What do the square brackets mean here? These settings are supposed to be yaml lists or something else?